llvm/test/CodeGen/X86/yk-insert-stackmaps.ll

```diff
@@ -0,0 +1,53 @@
+; RUN: llc -stop-after=yk-stackmaps --yk-insert-stackmaps < %s  | FileCheck %s
+
+; Check whether there is llvm.experimental.stackmaps section added after
```

Comment:
Minor nitpick: this it's not a section, but rather a call. So this should be 
```
Check whether `llvm.experimental.stackmap` calls are only inserted after normal calls, and not after external calls.
```

Reply:
Fixed: [2925973](https://github.com/ykjit/ykllvm/pull/114/commits/292597340984d5e844974e1c038133b373a8e3fe)